lib/source-map-consumer.js

```diff
@@ -951,21 +748,123 @@ function IndexedSourceMapConsumer(aSourceMap, aSourceMapURL) {
     }
     lastOffset = offset;
 
-    return {
-      generatedOffset: {
-        // The offset fields are 0-based, but we use 1-based indices when
-        // encoding/decoding from VLQ.
-        generatedLine: offsetLine + 1,
-        generatedColumn: offsetColumn + 1
-      },
-      consumer: new SourceMapConsumer(util.getArg(s, 'map'), aSourceMapURL)
-    }
+    const consumer = new SourceMapConsumer(util.getArg(s, 'map'), aSourceMapURL);
+    return consumer.then(consumer => {
+      return {
+        generatedOffset: {
+          // The offset fields are 0-based, but we use 1-based indices when
+          // encoding/decoding from VLQ.
+          generatedLine: offsetLine + 1,
+          generatedColumn: offsetColumn + 1
```

Comment:
Not introduced here, but elsewhere it says that columns are 0-based (what I recall as well), so I wonder if this is a latent bug.

Reply:
Perhaps? I had to do this to get the tests passing with the wasm. I think we can investigate further in a follow up.